src/maintainer/pinning_deps.rst

```diff
@@ -3,6 +3,20 @@
 Pinned dependencies
 *******************
 
+**Pinning a dependency** is a practice of explicitly pointing to the version of a library which our package is linked with. 
```

Comment:
```suggestion
**Pinning a dependency** is the practice of explicitly pointing to the version of a library which our package is linked with. 
```
I am making this comment solely in my personal capacity and am not conveying any rights to any intellectual property of any third parties.